### PR TITLE
Xcmv3 Decorator

### PR DESCRIPTION
**Pull request by mkchungs**
Disable xcmmsg.assetChains in favor of performAnalysisInstructions
 

Test cases:

* [queryResponse] https://polkaholic.io/xcmmessage/0x0242e6ee3b84d4004b09b85cac75adb335fa2969a956434cb46f010ef9ac2665/17335451
* [queryResponse] https://polkaholic.io/xcmmessage/0xfa761ea41ea4e649be13486941d469d17567aed5efb4dbf5a7ab28617c82e009/17335451
[receiveTeleportedAsset V3] https://polkaholic.io/xcmmessage/0xd772b7b0e5a730d7dbe921ffe6eb402e5abe7bec49e904adb498cd5936106cb3/17341726
* [queryResponse] https://polkaholic.io/xcmmessage/0xd75428f6450277a68e01a642bdb5f8537b6f31a42f5b8786267e55dc6b952594/17336080

### CODE REVIEW

File: substrate/query.js [hunk 1]
```diff
@@ -5793,6 +5809,7 @@ module.exports = class Query extends AssetManager {
         let version = dXcmMsg.version
         let dInstructionV = {}
         switch (instructionK) {
+            case "reserveAssetDeposited":
```
Comment (sourabhniyogi) [inline substrate/query.js:5812]: receiveTeleportedAsset

File: substrate/query.js [hunk 2]
```diff
@@ -5892,27 +6007,48 @@ module.exports = class Query extends AssetManager {
...
+                    for (let i = 0; i < xcmMsgV.length; i++) {
+                        let instructionK = Object.keys(xcmMsgV[i])[0]
+                        xcmPath.push(instructionK)
+                    }
+                    //console.log(`decorateXCMMsg Path`, xcmPath)
+                    for (let i = 0; i < xcmPath.length; i++) {
+                        let v2_instructionK = xcmPath[i]
+                        let v2_instructionV = xcmMsgV[i][v2_instructionK]
+                        //console.log(`v2_instructionK=${v2_instructionK}, v2_instructionV`, v2_instructionV)
+                        await this.decorateXCMInstructionV3(chainID, chainIDDest, dXcmMsg, v2_instructionK, v2_instructionV, blockTS, dAssetChains, decorate, decorateExtra)
```
Comment (sourabhniyogi) [inline substrate/query.js:6032]: v2

File: substrate/query.js [hunk 3]
```diff
@@ -5877,6 +5894,104 @@ module.exports = class Query extends AssetManager {
...
+                    }
+                }
+                console.log(`depositReserveAsset final`, JSON.stringify(instructionV, null, 4))
+                dInstructionV[instructionK] = instructionV
+                dXcmMsg[version].push(dInstructionV)
+                break;
+            case "transact":
+                if (chainIDDest != undefined) {
+                    //init with chainDest
+                    await this.setupQuerChainAndAPI(chainIDDest);
```
Comment (sourabhniyogi) [inline substrate/query.js:5966]: setupQueryChainAndAPI

**Comment by mkchungs** [comment]
Fixed

**Pull request merged**